### ISSUE TITLE
FLASH sector description is wrong for NUCLEO F722ZE

### ISSUE THREAD

**Issue by jeromecoutant** [STMicroelectronics/STM32CubeF7#56]
Hi

FLASH sector description is wrong for NUCLEO F722ZE:
https://github.com/STMicroelectronics/STM32CubeF7/blob/master/Projects/STM32F722ZE-Nucleo/Examples/FLASH/FLASH_EraseProgram/Inc/main.h#L33-L44

I propose:
```
#define ADDR_FLASH_SECTOR_0     ((uint32_t)0x08000000) /* Base @ of Sector 0, 16 Kbytes */
#define ADDR_FLASH_SECTOR_1     ((uint32_t)0x08004000) /* Base @ of Sector 1, 16 Kbytes */
#define ADDR_FLASH_SECTOR_2     ((uint32_t)0x08008000) /* Base @ of Sector 2, 16 Kbytes */
#define ADDR_FLASH_SECTOR_3     ((uint32_t)0x0800C000) /* Base @ of Sector 3, 16 Kbytes */
#define ADDR_FLASH_SECTOR_4     ((uint32_t)0x08010000) /* Base @ of Sector 4, 64 Kbytes */
#define ADDR_FLASH_SECTOR_5     ((uint32_t)0x08020000) /* Base @ of Sector 5, 128 Kbytes */
#define ADDR_FLASH_SECTOR_6     ((uint32_t)0x08040000) /* Base @ of Sector 6, 128 Kbytes */
#define ADDR_FLASH_SECTOR_7     ((uint32_t)0x08060000) /* Base @ of Sector 7, 128 Kbytes */
```

**Comment by RKOUSTM** [comment]
Hi @jeromecoutant,

Thank you for your contribution. An internal tracker has been logged and a fix will be implemented and made available in the frame of a future release.

Thank you once again for your contribution. 

With regards,
Rihab

**Comment by RKOUSTM** [comment]
Hi @jeromecoutant,

I hope you are fine. The issue you reported has been fixed in the frame of version v1.16.2 of the STM32CubeF7 published on GitHub. Thank you again for having reported.

With regards,
